docs/source/concepts/rate-controllers.md

```diff
@@ -285,13 +285,36 @@ The template can (**and should**) contain special “variables/placeholders” t
 ...
 ```
 
+The value of `i` starts at **0**, so the **first line** represents **0 transactions submitted**.  
+Each value in a line represents **time in milliseconds** from the start of the round. 
+
 **Binary format**: Both binary representations encode the `X` number of recordings as a series of `X+1` UInt32 numbers (1 number for the array length, the rest for the array elements), either in Little Endian or Big Endian encoding:
 
 ```sh
 Offset: |0      |4      |8      |12      |16      |...     
 Data:   |length |1st    |2nd    |3rd     |4th     |...      
 ```
 
+- The first entry (length) indicates the number of transactions submitted.
+- Each subsequent entry in the array represents the timestamp of a transaction being enabled (submitted) by the rate controller.
+- The subsequent values (Uint32 values in Big Endian or Little Endian format) represent time in milliseconds from the start of the round. Each entry corresponds to the number of transactions submitted starting from 0
+
+For example, if there are 3 transactions enabled, the binary format would look like:
+
+```sh
+Offset: |0      |4      |8      |12      
+Data:   |length |1st    |2nd    |3rd
+        |3      |100    |200    |300
+```
+
+If no transactions are submitted, it would look like:

```

Comment:
This isn't correct, so please remove the section starting "If no transactions are submitted, it would look like:"